dashboards/aws-marketplace/aws-marketplace-spg.yaml

```diff
@@ -0,0 +1,270 @@
+              ColumnName: usage_type
+              NewColumnName: Usage Type
+          - RenameColumnOperation:
+              ColumnName: subscription_start_date
+              NewColumnName: Subscription Start Date
+          - RenameColumnOperation:
+              ColumnName: subscription_end_date
+              NewColumnName: Subscription End Date
+          - RenameColumnOperation:
+              ColumnName: operation
+              NewColumnName: Operation
+          - RenameColumnOperation:
+              ColumnName: item_description
+              NewColumnName: Item Description
+          - RenameColumnOperation:
+              ColumnName: availability_zone
+              NewColumnName: Availability Zone
+          - RenameColumnOperation:
+              ColumnName: region
+              NewColumnName: Region
+          - RenameColumnOperation:
+              ColumnName: legal_entity
+              NewColumnName: Legal Entity
+          - RenameColumnOperation:
+              ColumnName: billing_entity
+              NewColumnName: Billing Entity
+          - RenameColumnOperation:
+              ColumnName: pricing_unit
+              NewColumnName: Pricing Unit
+          - RenameColumnOperation:
+              ColumnName: resource_id_count
+              NewColumnName: Resource Id Count
+          - RenameColumnOperation:
+              ColumnName: usage_quantity
+              NewColumnName: Usage Quantity
+          - RenameColumnOperation:
+              ColumnName: unblended_cost
+              NewColumnName: Unblended Cost
+          - RenameColumnOperation:
+              ColumnName: amortized_cost
+              NewColumnName: Amortized Cost
+          Source:
+            PhysicalTableId: 90e899f8-a44f-478a-8f04-5a20a149a2c8
+      ImportMode: SPICE
+    dependsOn:
+      views:
+      - account_map
+      - marketplace_summary_view
+    schedules:
+    - default
+views:
+  marketplace_summary_view:
+    dependsOn:
+      cur: true
+    data: |-
+      CREATE OR REPLACE VIEW ${athena_database_name}.marketplace_summary_view AS
+      SELECT
+        "year" "year"
+      , "month" "month"
+      , "bill_billing_period_start_date" "billing_period"
+      , "date_trunc"('day', "line_item_usage_start_date") "usage_date"
+      , "bill_payer_account_id" "payer_account_id"
+      , "line_item_usage_account_id" "linked_account_id"
+      , "bill_invoice_id" "invoice_id"
+      , "line_item_line_item_type" "charge_type"
+      , (CASE WHEN ("line_item_line_item_type" = 'DiscountedUsage') THEN 'Running_Usage' WHEN ("line_item_line_item_type" = 'SavingsPlanCoveredUsage') THEN 'Running_Usage' WHEN ("line_item_line_item_type" = 'Usage') THEN 'Running_Usage' ELSE 'non_usage' END) "charge_category"
+      , (CASE WHEN ("savings_plan_savings_plan_a_r_n" <> '') THEN 'SavingsPlan' WHEN ("reservation_reservation_a_r_n" <> '') THEN 'Reserved' WHEN ("line_item_usage_type" LIKE '%Spot%') THEN 'Spot' ELSE 'OnDemand' END) "purchase_option"
```

Comment:
Does it applicable for marketplace? You can drop it I think

Reply:
purchase_option is not needed.